man/org.freedesktop.login1.xml

```diff
@@ -672,16 +672,17 @@ node /org/freedesktop/login1 {
 #define SD_LOGIND_SKIP_INHIBITORS                (UINT64_C(1) &lt;&lt; 4)
       </programlisting>
       <para>When the <varname>flags</varname> is 0 then these methods behave just like the versions without
-      flags. Since systemd version 256 <constant>SD_LOGIND_ROOT_CHECK_INHIBITORS</constant> (0x01) is deprecated,
-      and active inhibitors are always honoured by default for privileged users too, and a new flag
-      <constant>SD_LOGIND_SKIP_INHIBITORS</constant> (0x04) can be specified to bypass inhibitors. When
-      <constant>SD_LOGIND_KEXEC_REBOOT</constant> (0x02) is set, then <function>RebootWithFlags()</function>
-      performs a kexec reboot if kexec kernel is loaded. When <constant>SD_LOGIND_SOFT_REBOOT</constant>
-      (0x04) is set, or <constant>SD_LOGIND_SOFT_REBOOT_IF_NEXTROOT_SET_UP</constant> (0x08) is set and a
-      new root file system has been set up on <literal>/run/nextroot/</literal>, then
-      <function>RebootWithFlags()</function> performs a userspace reboot only.
-      <constant>SD_LOGIND_SOFT_REBOOT_IF_NEXTROOT_SET_UP</constant> and
-      <constant>SD_LOGIND_KEXEC_REBOOT</constant> can be combined, with soft-reboot having precedence.</para>
+      flags. Since systemd version 257 active inhibitors are always honoured by default for privileged users
+      too, and <constant>SD_LOGIND_ROOT_CHECK_INHIBITORS</constant> (0x01) only applies to weak inhibitors,
```

Comment:
you should say what SD_LOGIND_ROOT_CHECK_INHIBITORS does, not just where it applies